06.bowling_object/game.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require './shot'
+require './frame'
+
+class Game
+  def initialize(score)
+    @frames = parse_frames(score)
+  end
+
+  def calc
+    total_score = 0
+    @frames.each_with_index do |frame, frame_count|
+      total_score += frame.score(@frames, frame_count)
```

Comment:
何番目のフレームなのかは引数で渡すより、フレーム自身が「僕はn番目です」というのを知っている状態にした方がベターだと思います。

```suggestion
      total_score += frame.score(@frames)
```

Reply:
フレームのインスタンス変数に何番目かを入れるようにしました